app/Http/Requests/ContactRequest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class ContactRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, mixed>
+     */
+    public function rules()
+    {
+        return [
+            'name'=> 'required', 'min:1',
+            'email'=> 'required', 'email:rfc',
+            'phone'=> 'required', 'phone:AUTO,US',
+            'text'=> 'required', 'min:1'
```

Comment:
что-то не то) так не работает

вот так должно быть
```suggestion
            'name'=> ['required', 'min:1'],
            'email'=> ['required', 'email:rfc'],
            'phone'=> ['required', 'phone:AUTO,US'],
            'text'=> ['required', 'min:1'],
```

или

```suggestion
            'name'=> 'required|min:1',
            'email'=> 'required|email:rfc',
            'phone'=> 'required|phone:AUTO,US',
            'text'=> 'required|min:1'
```